### PR TITLE
data: ronthetech

### PR DESCRIPTION
**Pull request by ronthetech**
🚀

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4858"></a>

### CODE REVIEW

File: data/ronthetech.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Connect with me on LinkedIn",
+			"url": "https://www.linkedin.com/in/ronjf/",
+			"icon": "FaLinkedin"
+		},
+		{
+			"name": "Github",
+			"url": "https://www.github.com/ronthetech",
```
Comment (Panquesito7) [inline data/ronthetech.json:17]: Non-blocking: not sure the `www.` is 100% needed here.
```suggestion
			"url": "https://github.com/ronthetech",
```

**Review by Panquesito7 — DISMISSED** [review]
Otherwise, it's looking good. Thanks! 🎉

**Comment by ronthetech** [comment]
Thanks for the suggestion!

**Review by Panquesito7 — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**